content/docs/accessibility.md

```diff
@@ -104,83 +105,82 @@ function ListItem({ item }) {
 }
 ```
 
-For more info, see [the Fragments documentation](/docs/fragments.html).
+Մանրամասների համար նայեք [Fragment-ների փաստաթուղթը](/docs/fragments.html)։
 
-## Accessible Forms {#accessible-forms}
+## Հասանելի form-եր {#accessible-forms}
 
-### Labeling {#labeling}
-Every HTML form control, such as `<input>` and `<textarea>`, needs to be labeled accessibly. We need to provide descriptive labels that are also exposed to screen readers.
+### Անվանադրություն {#labeling}
+Յուրաքանչյուր HTML form-ի կառավարում, ինչպիսիք են `<input>` կամ `<textarea>`, պահանջում են հասանելի անվանում։ Մենք պետք է տրամադրենք բնութագրող անվանումներ, որոնք կտրամադրվեն նաև էկրան կարդացողներին։
 
-The following resources show us how to do this:
+Հետևյալ ռեսուրսները ցույց են տալիս, թե ինչպես\`
 
-- [The W3C shows us how to label elements](https://www.w3.org/WAI/tutorials/forms/labels/)
-- [WebAIM shows us how to label elements](https://webaim.org/techniques/forms/controls)
-- [The Paciello Group explains accessible names](https://www.paciellogroup.com/blog/2017/04/what-is-an-accessible-name/)
+- [W3C֊ը ցույց է տալիս ինչպես անվանադրել էլեմենտները](https://www.w3.org/WAI/tutorials/forms/labels/)
```

Comment:
- [W3C-ը ցույց է տալիս ինչպես անվանադրել էլեմենտները](https://www.w3.org/WAI/tutorials/forms/labels/)<sub>`eng`</sub>
- [WebAIM-ը ցույց է տալիս ինչպես անվանադրել էլեմենտները](https://webaim.org/techniques/forms/controls)<sub>`eng`</sub>
- [Paciello Group-ը բացատրում է հասանելի անունները](https://www.paciellogroup.com/blog/2017/04/what-is-an-accessible-name/)<sub>`eng`</sub>